packages/portfolio-contract/tools/plan-solve.ts

```diff
@@ -64,7 +64,11 @@ export type LpModel = IModel<string, string>;
  *   specified chain
  */
 export type GasEstimator = {
-  getWalletEstimate: (chainName: AxelarChain) => Promise<bigint>;
+  getWalletEstimate: (
+    chainName: AxelarChain,
+    operationType?: 'supply' | 'withdraw' | 'depositforburn',
+    protocol?: 'aave' | 'compound',
```

Comment:
```suggestion
    protocol?: YieldProtocol,
```
(from portfolio-api/src/constants)

Reply:
@gibson042 even if it's always a proper subset of?

Reply:
Made the change.